plugins/fortplan.cpp

```diff
@@ -88,7 +102,7 @@ DFhackCExport command_result plugin_init ( color_ostream &out, vector <PluginCom
 #define DAY_TICKS 1200
 DFhackCExport command_result plugin_onupdate(color_ostream &out)
 {
-    if (Maps::IsValid() && !World::ReadPauseState() && world->frame_counter % (DAY_TICKS/2) == 0)
+    if (Maps::IsValid() && !World::ReadPauseState() && df::global::world->frame_counter % (DAY_TICKS/2) == 0)
```

Comment:
One existing convention is to use `REQUIRE_GLOBAL(world)`, usually near the `DFHACK_PLUGIN` call, which imports the specified global into the global namespace and also tracks which globals a plugin uses. In practice, recent DFHack versions have all globals available, but using REQUIRE_GLOBAL makes it a bit safer, so I would recommend that. Keeping the `df::global::` prefix isn't strictly necessary with it, but it's fine to have it here. I'm just bringing it up because I noticed it in the diff. (Did the reorganization break some code that was compiling previously?)

Reply:
added the REQUIRE_GLOBAL() calls to fortplan. the reorg removed the `using namespace` lines that were in buildingplan-lib.h (`using namespace` in header files can lead to some tricky bugs). I could have added them back in fortplan.cpp, but I just went with the long form since adding a `using namespace df::global` wouldn't have saved much.

Reply:
Yeah, I think the original buildingplan plugin was split up into a couple files in #464 for fortplan's purposes - at the time, it was more intended to be shared code than a proper library. Thanks for cleaning it up!